dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_29__Add_createdatclient_to_relationship.sql

```diff
@@ -0,0 +1,3 @@
+alter table relationship add column if not exists "createdatclient" timestamp;
+
+update relationship set createdatclient = created where createdatclient IS NULL;
```

Comment:
why do we set it to 'created'?

Reply:
I think it will be better to leave it null. Sometimes this field is used to tell if the data is coming from Android or not (usually is is null for web, not null for Android)